Docs/sphinx_documentation/source/GPU.rst

```diff
@@ -200,28 +246,60 @@ For example, on Cori GPUs you can specify the architecture as follows:
    cmake [options] -DENABLE_CUDA=yes -DCUDA_ARCH=Volta /path/to/amrex/source
 
 
-Note that AMReX only supports GPU architectures with version number ``6.0`` or higher.
+If no architecture is specified, CMake will try to determine which GPU architecture is
+supported by the system.
+If more than one is found, CMake will build for all of them.
+This will generally results in a larger library and longer build times.
+If autodetection fails, a set of "common" architectures is assumed.
+**Note that AMReX only supports GPU architectures with version number ``6.0`` or higher**.
```

Comment:
I think we also just support CUDA 9.0 and newer. Let's add this as well.

Reply:
Actually, it is still the case.

Reply:
```suggestion
**Note that AMReX only supports GPU architectures with compute capability ``6.0`` or higher**.
```

Reply:
NVIDIA